packages/editor/src/ide.ts

```diff
@@ -61,33 +62,42 @@ export class JsonEditorIde extends HTMLElement implements Editor {
       this.editor = document.createElement('json-forms') as JsonFormsElement;
       this.appendChild(this.editor);
     }
-    this.editor.setInnerComponent(
-      TreeRenderer,
-      {
-        uischema: getUiSchema(this._store.getState()),
-        schema: getSchema(this._store.getState()),
-        schemaService
-      });
-    this.editor.store = this._store;
 
-    const exportButton = document.getElementById('export-data-button') as HTMLButtonElement;
-    if (exportButton !== null) {
-      const exportDialog = createExportDataDialog();
-      document.body.appendChild(exportDialog);
-      configureExportButton(this, exportButton, exportDialog);
-    }
+    JsonRefs.resolveRefs(getSchema(this._store.getState()))
+      .then(
+        resolvedSchema => {
+        this.editor.setInnerComponent(
+          TreeRenderer,
+          {
+            uischema: getUiSchema(this._store.getState()),
+            schema: getSchema(this._store.getState()),
+            schemaService,
+            resolved: resolvedSchema.resolved
```

Comment:
Don't we always want to work with the resolved schema? So what happens if we use `resolvedSchema.resolved` as `schema`, do we still need the `resolved` property?

Reply:
I don't think so, they are same so we can directly use `resolvedSchema.resolved` as our root schema

Reply:
Could you check if that's possible and update if that's the case?